### PR TITLE
Update: add mpc paper

### PR DESCRIPTION
**Pull request by BeStrongok**
**Type of change**

- [x] Add new papers (Please tell us why you think this paper is awesome!)
- [ ] Fix the category of an existing paper/papers (Please tell us the reasons)
- [ ] Add a new tool/primitive/application with a new markdown page (Thank you! Also, please tell us more about this awesome thing!)

**Description**

> Please include a summary of the change and the relevant motivation.

Add a new **Overdrive LowGear 2.0** paper: 
this paper presents a new offline phase based on linear homomorphic encryption that reduces the bandwidth and number of communications rounds compared to the state-of-the-art MPC protocol for a small number of parties, as well as matrix triples generation, which i think it could be utilized in low bandwidth scenarios.

### CODE REVIEW

**Comment by BeStrongok** [comment]
**_I have read the CLA Document and I hereby sign the CLA_**

File: docs/awesome-pets/papers/tools/mpc.md [hunk 1]
```diff
@@ -155,6 +155,12 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
   *Elette Boyle, Geoffroy Couteau, Niv Gilboa, Yuval Ishai, Lisa Kohl, Nicolas Resch, Peter Scholl*
   Crypto 2022, [eprint](https://eprint.iacr.org/2022/1014), BCG+22
 
+###  Linear Homomorphic Encryption
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/tools/mpc.md:158]: Would you please tell us the reason for adding this section?

Comment (Jamie-Cui) [reply docs/awesome-pets/papers/tools/mpc.md:158]: One more suggestion: could you rename the section title to `others`? :P

**Comment by Jamie-Cui** [comment]
Hi, there's merge conflicts in this PR, would you please resolve it? Many thanks :P

**Pull request closed without merging**